tests/unit/load-webpack.ts

```diff
@@ -0,0 +1,145 @@
+import * as assert from 'intern/chai!assert';
+import * as registerSuite from 'intern!object';
+import global from '../../src/global';
+import load from '../../src/load-webpack';
+
+interface WebpackModules {
+	[id: number]: any;
+}
+
+let webpackModules: WebpackModules;
+function setModules(modules?: { [mid: string]: any }) {
+	let nextId = 0;
+	webpackModules = Object.create(null) as WebpackModules;
+
+	if (modules) {
+		const idMap = Object.create(null);
+		Object.keys(modules).forEach((mid: string) => {
+			const id = nextId++;
+			idMap[mid] = { id, lazy: mid.indexOf('bundle!') > -1 };
```

Comment:
Also, is this an issue that should be resolved before the PR lands, or should/can it be postponed for an ongoing discussion?